src/writer/blocks/writerchatreplywithtoolconfig.py

```
@@ -0,0 +1,399 @@
+
+                    tool = writer.ai.FunctionTool(
+                        type="function",
+                        name=tool_name,
+                        description=description,
+                        callable=self._make_callable(tool_name, app_id, function_name),
+                        parameters=parameters or {},
+                    )
+                    tools.append(tool)
+                        
+                except Exception as e:
+                    continue
```

Ruff (0.14.3)

`try`-`except`-`continue` detected, consider logging the exception

(S112)